packages/@react-aria/searchfield/docs/useSearchField.mdx

```diff
@@ -83,14 +83,14 @@ function SearchField(props) {
   let {labelProps, inputProps} = useSearchField(props, state, ref);
 
   return (
-    <div style={{display: 'flex', flexDirection: 'column', width: 200}}>
+    <div style={{display: 'flex', direction: 'column', width: 200}}>
```

Comment:
Maybe add some padding between the label and the input here:
![image](https://user-images.githubusercontent.com/9661127/84960520-ac54da80-b0b6-11ea-9305-8d3d6937aac3.png)